### PR TITLE
Changed versions.py to fix Windows ValueError: path is on drive d: St…

### PR DESCRIPTION
**Pull request by timmyjl12**
Changed versions.py to fix Windows ValueError: path is on drive d: Start on drive C:

### CODE REVIEW

File: flask_debugtoolbar/panels/versions.py [hunk 1]
```diff
@@ -3,13 +3,15 @@
...
 from flask_debugtoolbar.panels import DebugPanel
+from flask import current_app
 
 _ = lambda x: x
 
 
 def relpath(location, python_lib):
     location = os.path.normpath(location)
-    relative = os.path.relpath(location, python_lib)
+    # relative = os.path.relpath(location, python_lib)
```
Comment (jeffwidman) [inline flask_debugtoolbar/panels/versions.py]: Curious why you commented this out rather than removing it?

**Comment by jeffwidman** [comment]
Oh, I see, this is an attempt to fix #132

**Comment by nanoric** [comment]
I think there are only two reason to use relative path:
 * security consideration. Using an abslolute path will leave too many information for attackers. But since "DEBUG" mode are usually only triggered in local or in developing environment. In my view, debug bar should not consider security issues. Because If someone installed debug tools in production environment and runs it in public, this guy must have made more serious security mistakes.
 * relative path is shorter and easier to read. Since almost all packages are installed at a same location, using relative path makes people easier to read. And I think this is why versions.py uses relpath.

So in conclusion, in my view, using relative path is better, because it is easier to read. And a try catch is necessary, if ValueError is thrown, use abslolute path instead. 
# 130

File: flask_debugtoolbar/panels/versions.py (hunk 1 above)
Comment (timmyjl12) [reply flask_debugtoolbar/panels/versions.py]: Hi Jeff, you're right I should have just removed it. I left that in for my troubleshooting purposes, but forgot to remove it. My apologies.

**Comment by timmyjl12** [comment]
Hi @jeffwidman this is a simple fix I used for operating code that may not be on your main volume. An example is your python path is on C:\, while your code lives on a different drive such as D:\. I was running into an error, so this was my simple fix. I found it easy to call the current app path so there was no longer an error and this would resolve the code correctly without an exception. Thank you.

**Comment by mgood** [comment]
Yeah, the relative path is just to be more readable here. As @nanoric mentioned, #132 is close but it should just return the absolute path if it can't compute a shorter relative path.
Though I noticed there's also `flask_debugtoolbar.utils.format_fname` which handles the Windows error, and is probably a nicer replacement for this implementation of `relpath`.

**Comment by macnewbold** [comment]
I know it's been a long time, but do you still have any interest in this?
If so, it will need to catch it up with master and retest. If not, we'll probably close it out.

**Comment by timmyjl12** [comment]
Hey @macnewbold, I'm going to suggest closing this. Thank you.
